src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -181,7 +182,20 @@ protected function describeContainerServices(ContainerBuilder $builder, array $o
      */
     protected function describeContainerDefinition(Definition $definition, array $options = array())
     {
-        $output = '- Class: `'.$definition->getClass().'`'
+        $output = '';
+        if (interface_exists('phpDocumentor\Reflection\DocBlockFactoryInterface')) {
+            try {
+                $reflectionProperty = new \ReflectionClass($definition->getClass());
+                $classDescription = DocBlockFactory::createInstance()->create($reflectionProperty->getDocComment())->getSummary();
+
+                if (!empty($classDescription)) {
+                    $output = '- Description: `'.$classDescription.'`'."\n";
```

Comment:
i would put the description after the class entry, given it acts as a title sort of

Reply:
Did not agree. I think the description is higher than class. The values are not sorted by alpha. I think they are sorted by prio.

```
...
        $output .= '- Class: `'.$definition->getClass().'`'
            ."\n".'- Public: '.($definition->isPublic() && !$definition->isPrivate() ? 'yes' : 'no')
            ."\n".'- Synthetic: '.($definition->isSynthetic() ? 'yes' : 'no')
            ."\n".'- Lazy: '.($definition->isLazy() ? 'yes' : 'no')
            ."\n".'- Shared: '.($definition->isShared() ? 'yes' : 'no')
            ."\n".'- Abstract: '.($definition->isAbstract() ? 'yes' : 'no')
            ."\n".'- Autowired: '.($definition->isAutowired() ? 'yes' : 'no')
            ."\n".'- Autoconfigured: '.($definition->isAutoconfigured() ? 'yes' : 'no')
...
```